### ISSUE TITLE
[Bug Report] Version 2.2.8 `page`  param donsen't  work

### ISSUE THREAD

**Issue by huangshuwei** [pipwerks/PDFObject#277]
### Browser 
Chrome 106.0.5249.119

### library Version
v2.2.8

### Step to reproduce
1.Set page by `page` param
```
var options = {
page: "2"
};
PDFObject.embed("/pdf/sample-3pp.pdf", "#my-container", options);
```

### What is actually happening?
 `page`  param donsen't  work

### Any additional comments
v2.2.7 works

**Comment by theodore-s-beers** [comment]
This appears to be a duplicate of #272.

**Comment by pipwerks** [comment]
Yes, dupe of #272. Fixed with 2.2.9 branch (currently in testing). Thanks
